public/stylesheets/scss/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$font-light: #B4C4CF;
+$font-lighter: #F3F5F7;
+$bg-light: #B4C4CF;
+$bg-lighter: #F3F5F7;
+$btn-darker: #0C3A45;
```

Comment:
Same here, I would rather use elements, it becomes like the stylesheet with time :P